pkg/filters/int.go

```diff
@@ -53,57 +67,89 @@ func (f *IntFilter) Enabled() bool {
 }
 
 func (f *IntFilter) Minimum() int64 {
-	return f.GreaterThan
+	return f.greaterThan
 }
 
 func (f *IntFilter) Maximum() int64 {
-	return f.LessThan
+	return f.lessThan
 }
 
-func (filter *IntFilter) Parse(operatorAndValues string) error {
-	if len(operatorAndValues) < 2 {
-		return fmt.Errorf("invalid operator and/or values given to filter: %s", operatorAndValues)
+// priority goes by (from most significant):
+// 1. equality
+// 2. greater
+// 3. lesser
+// 4. non equality
+func (f *IntFilter) Filter(val int64) bool {
+	result := !f.enabled || f.equal[val] || val > f.greaterThan || val < f.lessThan
```

Comment:
What about the case that the user want `val > x && val` < y ?

Reply:
Should we support that? If `x < y` then we're good anyway, on the other hand it's just a conflict, which means we either break the filter or, as is done here, and I remember adding a test demonstrating it, that case is done as `val > x || val < y`.
As we've talked, these filters are meant to include as much as possible for all requests, so a conflicting request won't and shouldn't override or break the filter.